### PR TITLE
Add Alfresco Community module

### PR DESCRIPTION
**Pull request by aborroy**

### CODE REVIEW

File: modules/alfresco/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Alfresco
+categories:
+  - other
+docs:
+  - id: java
+    url: https://java.testcontainers.org/modules/alfresco/
```
Comment (mdelapenya) [inline modules/alfresco/index.md:7]: Is this URL correct? I think it should point to the community site where the code/docs live, which is not the Java site. Could you update it with the right link 🙏 ?

Comment (aborroy) [reply modules/alfresco/index.md]: You're completely right. Now it's fixed.

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks!

File: modules/alfresco/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+title: Alfresco
+categories:
+  - other
+docs:
+  - id: java
+    url: https://github.com/AlfrescoLabs/alfresco-testcontainers
+    maintainer: community
+    example: |
+      ```java
+      var alfresco = new AlfrescoContainer("23.2.1");
```
Comment (eddumelendez) [inline modules/alfresco/index.md:11]: Let's use the full image for visibility

```suggestion
      var alfresco = new AlfrescoContainer(DockerImageName.parse("alfresco/alfresco-content-repository-community:23.2.1"));
```

**Review by eddumelendez — APPROVED** [review]

**Pull request merged**